ccm_web/server/test/app.e2e-spec.ts

```diff
@@ -0,0 +1,37 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { INestApplication } from '@nestjs/common'
+import request from 'supertest'
+import { AppModule } from './../src/app.module'
```

Comment:
There is no AppModule, this code seems to be unused but the import should be corrected atleast for confusion later

Reply:
OK, I got confused. What is the purpose of the BASE module? to me right now it serves the purpose for API routing. Can we rename it to API?

Reply:
I can re-name it to API if you'd like. Calling it an API when we'll likely have other API modules seemed kinda strange, but I'll change it.

Reply:
Done. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/68/commits/dbd80aedc2698eb5e67a1b70d0762e372c9cf2e9

Reply:
OK, I see what you mean. Can we have all the other API modules like merge section, create bulk user under this?